src/Runtime/Distributions/Automata/RegexpTreeBuilder.cs

```diff
@@ -80,50 +80,57 @@ private enum EdgeTypes
                 return BuildCompactRegex(automaton, collapseAlternatives);
             }
 
-            var condensation = automaton.ComputeCondensation(automaton.Start);
-
-            RegexpTreeNode<TElement, TElementDistribution>[][,] componentRegexps = Util.ArrayInit(
-                condensation.ComponentCount, i => BuildStronglyConnectedComponentRegexp(condensation.GetComponent(i)));
-            RegexpTreeNode<TElement, TElementDistribution>[] stateRegexps = Util.ArrayInit(
-                automaton.States.Count, i => RegexpTreeNode<TElement, TElementDistribution>.Nothing());
-
-            // Dynamic programming in reverse topological order
-            for (int componentIndex = 0; componentIndex < condensation.ComponentCount; ++componentIndex)
+            using (var condensation = automaton.ComputeCondensation(automaton.Start))
```

Comment:
Are all changes in this file indent shifts induced by `using condensation` blocks (and additions of said blocks)? Github's diff here is unreadable.

Reply:
Yes. I really want C# with it's using statements here :)

Reply:
I resolved, since the only reason for diff is the using block.